pkg/utils/common.go

```diff
@@ -614,29 +614,22 @@ func PrintUnreportedErrors(
 
 func NewLogger(
 	logName string,
-	logLevelStr string,
-	defaultLogLevelStr string,
+	aggregatorLogLevel string,
```

Comment:
I believe we may want to keep this function a bit more generic, not so aggregator oriented.
It may be possible to use it on another packages. Probably it can be used to set up main app 
log with a bit of refactor. For the moment lets keep the previos more generic arg names

Reply:
abstracted into NewSignatureAggregatorLoggerNewLogger